assets/src/application/ui/input/CurrencyInput/index.tsx

```diff
@@ -10,10 +10,10 @@ interface CurrencyInputProps {
 	id: string;
 	amount: string | number;
 	placeholder?: string;
-	onConfirm?: (result?: { amount: string | number; id: string }) => void;
+	onChange?: (result?: { amount: string | number; id: string }) => void;
```

Comment:
tested this change and works well;
initially thought that it will be triggered on every keyup / input value change